### PR TITLE
Hej pull mig ;p

### PR DESCRIPTION
**Pull request by Hamsterpants**

### CODE REVIEW

**Review by HenrikSkindhoj — APPROVED** [review]
LGTM

File: report.md [hunk 1]
```diff
@@ -72,7 +72,55 @@
 ## Conclusion
 
 ### Summary
+This project involves the development of a comprehensive gambling website featuring games like poker, roulette, and coinflip. The frontend is built using React, TypeScript, and Vite, ensuring a responsive and engaging user experience. The backend leverages Quarkus with Java/Kotlin and PostgreSQL to handle game logic, user authentication, and secure data management.
```
Comment (muItilingualism) [inline report.md:75]: We dont use kotlin, just Java

Comment (muItilingualism) [reply report.md:75]: Needs a maven mention

File: report.md [hunk 2]
```diff
@@ -72,7 +72,55 @@
...
+
+A key focus of the project is the integration of DevOps practices, utilizing Docker for containerization and GitHub Actions for continuous integration and deployment. This approach facilitates frequent updates and efficient scaling, ensuring the platform remains robust and adaptable to user needs.
+
+The project also emphasizes security, employing OIDC for token-based authentication, and follows a modular architecture to enhance maintainability and scalability. Comprehensive testing strategies are in place to ensure the quality and stability of the application across all components.
```
Comment (muItilingualism) [inline report.md:79]: We dont use OIDC.

File: report.md [hunk 3]
```diff
@@ -72,7 +72,55 @@
...
 [Summarize the main points of the project.]
 
 ### Future Work
+1. Enhance Frontend Development:
+    - Apply advanced React patterns and hooks for efficient state management.
+    - Implement TypeScript generics and advanced type systems for improved code reliability.
+    - Integrate Redux or MobX for centralized state management in larger applications.
+
+2. Backend Security and Authentication:
+    - Implement OAuth 2.0 and OpenID Connect (OIDC) for enhanced authentication and authorization.
```
[the remark this answers is not in the document]
Comment (muItilingualism) [reply report.md:90]: And we dont use OIDC

File: report.md [hunk 4]
```diff
@@ -72,7 +72,55 @@
...
+    - Optimize Docker images for smaller sizes and faster build times.
+    - Implement Kubernetes for orchestration of containerized applications.
+    - Explore serverless architectures using AWS Lambda or Google Cloud Functions.
+
+7. Full-stack Application Development:
+    - Design a comprehensive full-stack application integrating frontend and backend components.
+    - Implement real-time features using WebSockets or Server-Sent Events.
+    - Develop a robust API gateway for managing and securing microservices.
+
+By focusing on these areas, the project will align closely with the course objectives, enhancing both the technical capabilities of the application and the skills of the development team.
```
Comment (muItilingualism) [inline report.md:119]: Make this less point / list based and separate into paragraphs with subheadings. See #1 as an example

File: report.md [hunk 5]
```diff
@@ -72,7 +72,55 @@
...
+
+### Short description of project management
+Project management in the context of software development involves coordinating the planning, execution, and delivery of projects to achieve specific goals efficiently. It encompasses the use of various tools and methodologies to manage resources, timelines, and risks effectively.
+
+With the learning objectives in mind, project management ensures that students can apply selected frameworks for both frontend and backend development, implement state management, and set up backend security. It also involves applying package managers, establishing CI/CD pipelines, and utilizing containerization for deployment. Ultimately, project management guides the design and implementation of a full-stack application, integrating all these elements to deliver a cohesive and functional product.
+
 [Mention any potential future improvements or follow-up work.]
```
Comment (muItilingualism) [inline report.md:126]: Remove

File: report.md [hunk 6]
```diff
@@ -72,7 +72,55 @@
...
+    - Implement real-time features using WebSockets or Server-Sent Events.
+    - Develop a robust API gateway for managing and securing microservices.
+
+By focusing on these areas, the project will align closely with the course objectives, enhancing both the technical capabilities of the application and the skills of the development team.
+
+### Short description of project management
+Project management in the context of software development involves coordinating the planning, execution, and delivery of projects to achieve specific goals efficiently. It encompasses the use of various tools and methodologies to manage resources, timelines, and risks effectively.
+
+With the learning objectives in mind, project management ensures that students can apply selected frameworks for both frontend and backend development, implement state management, and set up backend security. It also involves applying package managers, establishing CI/CD pipelines, and utilizing containerization for deployment. Ultimately, project management guides the design and implementation of a full-stack application, integrating all these elements to deliver a cohesive and functional product.
```
Comment (muItilingualism) [inline report.md:124]: "students" -> "we"

**Review by muItilingualism — CHANGES_REQUESTED** [review]
Review

File: report.md (hunk 5 above)
Comment (Hamsterpants) [reply report.md:126]: @muItilingualism

**Review by muItilingualism — CHANGES_REQUESTED** [review]
Review2

**Review by muItilingualism — APPROVED** [review]

**Pull request merged**